Make resource limits on lsst-login more obvious (#462)
lsst-dm/dm_dev_guide: services/lsst-login.rst

Overview
========

The ``lsst-login`` servers are primarily intended as bastions used to access other resources at NCSA.

> This node should only be used as a proxy for lsst-devl (see :doc:`/services/lsst-devl`) or submit batch jobs to perform work that requires more significant resources (see :doc:`/services/batch`).

Reviewer: ```suggestion
These servers should only be used as a jump host for the ``lsst-devl`` machines (see :doc:`/services/lsst-devl`) or to submit batch jobs to perform work that requires more significant resources (see :doc:`/services/batch`).
It may also be easier to use these machines than the ``lsst-devl`` machines with remote editors and IDEs.
```

> This node has resource limits that will cause larger tasks (such as compiling afw) to fail due to memory and other limits.

Reviewer: ```suggestion
These servers have resource limits, such as a small amount of memory per core, that will cause larger tasks (such as compiling ``afw``) to fail.
```
